shared/defs.proto

```diff
@@ -194,4 +194,32 @@ message ListProgress {
     float percentage = 2;
     sfixed32 checked_count = 3;
     sfixed32 total_count = 4;
-}
\ No newline at end of file
+}
+
+// parameters of a 2D Gaussian component for image fitting
+message GaussianComponent {
+    // x coordinate of the center in pixels
+    double center_x = 1;
```

Comment:
We have a sub-message `Point` for x,y pairs. I think we should use that, unless they specifically need to be double-precision? Even so, we could make a new message `DoublePoint`

Reply:
I think float would be sufficient. will change to `Point`